### DISCUSSION TITLE
Convert yaml to json files

### DISCUSSION THREAD

**Discussion by sergey-shpak** [parcel-bundler/parcel#8334] [General] [answered]
Hey guys,
Currently I'm using parcel to build web extension and want to store localization files using yaml format.
I'm totally ok to run one more process to convert yaml to json files and then run the main bundling process.
So, the question is - is there any possibility to convert yaml to json using parcel (any kind of configuration options, etc)?

**Comment by mischnic** [comment]
Like https://parceljs.org/languages/yaml/ ?

**Comment by sergey-shpak** [comment]
@mischnic not exactly, I need to convert all the files in directory from yaml into json format (without any usage in js) and later start another process that will include json files into final bundle (which is successfully done by @parcel/config-webextension)

*to give more context, localization files for web extensions are packaged and shipped in extension package and later browser loads those files and provides an API to access the content of those files: https://developer.chrome.com/docs/extensions/reference/i18n/

**Reply by mischnic** [reply]
So the real problem here is that you want `message.json` to be yaml files instead?

https://github.com/parcel-bundler/parcel/blob/62f64cae33a7bc1d2f5117472f9fd8bc2a931ab0/packages/transformers/webextension/src/WebExtensionTransformer.js#L86-L94

**Reply by sergey-shpak** [reply]
@mischnic yes, you got it right!

**Reply by sergey-shpak** [reply]
hey @mischnic any quick suggestions how to achieve this goal? 
or it would require additional parcel/webextension development?

**Reply by mischnic** [reply]
No, it's currently not possible without changes to the Parcel transformer.

**Comment by sergey-shpak** [comment, marked as answer]
@mischnic since it is no doable with configuration only, and it is no something that everyone needs - there is no much sense to change parcel transformer, so I'll go with my custom solution for that particular case, thanks for you help!
